src/components/Search/SearchContainer.module.scss

```diff
@@ -109,3 +109,20 @@
 .suggestion:hover .remove {
   display: block;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
Can be achieved using [`className="row justify v-center"`](https://github.com/santiment/san-webkit/blob/master/stories/Introduction.stories.mdx)